CheatSheets/markdown-cheatsheet.md

```diff
@@ -17,3 +17,11 @@
 | `1.first` or `2.Second`                 | Ordered List    |
 | `---`                                   | Horizontal line |
 | `![alt text](image.jpg)`                | Image           |
+| \`code\`                                | code            |
+| \~\~Strikethrough\~\~                   | Strikethrough   |
+| \#\#\# Heading ID                       | Heading         |
+| H\~2\~0                                 | Subscript       |
+| x\^2\^                                  | Superscript     |
```

Comment:
I have resolved this issue and why we need an HTML tag in the markdown cheatsheet as <sub></sub> is an HTML tag.

Reply:
Hi @sauravraw, 
Your pull request has `syntactical errors`. Please, follow the `actual syntax of markdown`. `<sub>` and `<sub>` both are also the markdown syntax for Subscript text and Superscript text.

Please, remove the `Heading`, because it already exists. 
Before modification, make sure your branch is `up-to-date`.